lib/secret_santa_api/secret_santa/party.ex

```diff
@@ -1,6 +1,11 @@
 defmodule SecretSantaApi.SecretSanta.Party do
   use Ecto.Schema
 
+  # Strips values from the struct before passing to Poison.Encoder
+  # to convert to JSON. Poison.Encoder is unable to parse private
+  # values such as __meta__.
+  @derive {Poison.Encoder,
```

Comment:
We may also want to explicitly say that we're also using this as a serializer to not show the end user attributes they shouldn't see.